alerting/src/main/kotlin/com/amazon/opendistroforelasticsearch/alerting/settings/DestinationMailSettings.kt

```diff
@@ -0,0 +1,68 @@
+package com.amazon.opendistroforelasticsearch.alerting.settings
+
+import org.elasticsearch.common.settings.Setting
+import org.elasticsearch.common.settings.Settings
+import org.elasticsearch.common.settings.SecureSetting
+import org.elasticsearch.common.settings.SecureString
+import java.io.IOException
+
+/**
+ * settings specific to mail destination.
+ */
+data class DestinationMailSettings(
+    val host: String,
+    val port: Int,
+    val method: String,
+    val from: String,
+    val username: SecureString,
+    val password: SecureString
+) {
+    companion object {
+        val DESTINATION_MAIL_HOST = Setting.simpleString(
+                "opendistro.alerting.destination.mail.host",
+                "localhost",
+                Setting.Property.NodeScope
+        )
+
+        val DESTINATION_MAIL_PORT = Setting.intSetting(
+                "opendistro.alerting.destination.mail.port",
+                587,
```

Comment:
Also, I think I was mistaken here, since port `25` is the default without authentication and you have authentication disabled by default, it would make sense for `25` to be the default.

I didn't test sending emails without authentication but I'm assuming it would expect port `25` in such a case.
(When I tested with authentication and `ssl` for example though it did need `465` to work).